testing/dart/window_hooks_integration_test.dart

```diff
@@ -125,65 +140,101 @@ void main() {
     test('onPointerDataPacket preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      PointerDataPacket data;
 
       runZoned(() {
         innerZone = Zone.current;
-        window.onPointerDataPacket = (_) {
+        window.onPointerDataPacket = (PointerDataPacket value) {
           runZone = Zone.current;
+          data = value;
         };
       });
 
-      _dispatchPointerDataPacket(new ByteData.view(new Uint8List(0).buffer));
+      final ByteData testData = new ByteData.view(new Uint8List(0).buffer);
+      _dispatchPointerDataPacket(testData);
       expect(runZone, isNotNull);
       expect(runZone, same(innerZone));
+      expect(data.data, equals(_unpackPointerDataPacket(testData).data));
     });
 
     test('onSemanticsEnabledChanged preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      bool enabled;
 
       runZoned(() {
         innerZone = Zone.current;
         window.onSemanticsEnabledChanged = () {
           runZone = Zone.current;
+          enabled = window.semanticsEnabled;
         };
       });
 
       _updateSemanticsEnabled(window._semanticsEnabled);
       expect(runZone, isNotNull);
       expect(runZone, same(innerZone));
+      expect(enabled, isNotNull);
+      expect(enabled, equals(window._semanticsEnabled));
     });
 
     test('onSemanticsAction preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      int action;
 
       runZoned(() {
         innerZone = Zone.current;
-        window.onSemanticsAction = (_, __) {
+        window.onSemanticsAction = (int value, __) {
```

Comment:
nit: reduce the number of underscores for second arg?

Reply:
Done.